addons/payment_razorpay/data/ir_cron_data.xml

```diff
@@ -0,0 +1,10 @@
+<odoo noupdate="1">
+    <record id="ir_cron_razorpay" model="ir.cron">
+        <field name="name">Razorpay Refresh token</field>
+        <field name="model_id" ref="model_payment_provider"/>
+        <field name="state">code</field>
+        <field name="code">model.cron_razorpay_refresh_token()</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
```

Comment:
```suggestion
    <record id="ir_cron_razorpay" model="ir.cron">
        <field name="name">Razorpay Refresh token</field>
        <field name="model_id" ref="model_payment_provider"/>
        <field name="state">code</field>
        <field name="code">model.cron_razorpay_refresh_token()</field>
        <field name="interval_number">1</field>
        <field name="interval_type">days</field>
```

Is `doall` not needed?
 [link ](https://www.odoo.com/documentation/master/developer/reference/backend/actions.html?highlight=numbercall)